src/App.css

```diff
@@ -54,18 +75,40 @@ nav a {
   color: var(--Black);
 }
 
-.booklist-container h1 {
-  text-align: center;
-}
+.booklist-container h1 { text-align: center; }
 
 .book {
   display: flex;
   width: 90%;
-  background: var(--Grey);
-  margin: 5px auto;
-  padding: 0.7rem 1.3rem;
+  background: var(--White);
+  margin: 1rem auto;
+  padding: 1.3rem;
```

Comment:
- [ ] The background color for the book cards should be white (`#fff`) however, I noticed that you specified `#eee` as white in your CSS variables. Kindly make this change so that your design looks closer to the Zeplin design template👍
- [ ] The background of your entire app should be `#fafafa` so that it is similar to the Zeplin design

## Your Book card
![image](https://github.com/christianonoh/book-town/assets/46839548/7c68bf18-4579-44cd-bfb4-cbc7c96cef80)
 
## Zeplin Book card
![image](https://user-images.githubusercontent.com/12745474/237868244-67519f48-79c1-4968-8afa-a2d89a5c9451.png)